src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,18 @@
 package core.basesyntax;
 
 public class Application {
+    private static final int NUMBER_OF_BALLS = 3;
+
     public static void main(String[] args) {
+        Lottery[] lotteries = new Lottery[NUMBER_OF_BALLS];
+        for (int i = 0; i < NUMBER_OF_BALLS; i++) {
+            lotteries[i] = new Lottery();
```

Comment:
It's better not to create several instances of the same class if you can use only one instance for your purposes, as stated [here](https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery#dont-create-several-instances-of-the-same-class-if-you-can-use-only-one-instance-for-your-purposes)